### ISSUE TITLE
updating catalyst versions while keeping namespace data

### ISSUE THREAD

**Issue by kooomix** [enigmampc/catalyst#464]
It is recommended [here](https://enigma.co/catalyst/install.html) to uninstall and then install rather than updating in order to avoid side effects of dependencies. But uninstalling looses namespace data which cause problems when running live or paper modes.

Amy suggestion how to properly update catalyst versions without losing namespace data?

**Comment by lenak25** [comment]
Hi @kooomix ,

> But uninstalling looses namespace data

Please elaborate on how you perform the uninstall procedure.

Thanks,

Lena

**Comment by kooomix** [comment]
$ source activate catalyst
$ pip uninstall enigma-catalyst
$ pip install enigma-catalyst

**Comment by lenak25** [comment]
Are you referring to the `~/.catalyst/data` directory?

**Comment by kooomix** [comment]
yes, assuming this is the only place an algorithm namespace data is being
logged.

**Comment by lenak25** [comment]
The live algorithms data is being saved under `~/.catalyst/data/live_algos/<algo_namespace>` which is not being deleted after uninstalling catalyst. 
Please describe the issue you are having when upgrading to a newer release.

Thanks,

Lena

**Comment by lenak25** [comment]
@kooomix are you still experience this issue? 
If so, please provide more information.

Thanks
